core/src/test/java/io/questdb/test/griffin/wal/WalPurgeJobTest.java

```diff
@@ -46,6 +46,10 @@
 import org.junit.Test;
 
 import java.io.File;
+import java.util.ArrayList;
```

Comment:
nit: please use our collections, ObjList() etc, for consistency

Reply:
Sure thing, I'll change it.